src/VirtoCommerce.News.Web/App/src/modules/vc-news/pages/news-article-list.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <VcBlade :title="$t('VC_NEWS.PAGES.LIST.TITLE')" width="50%" :expanded="expanded" :closable="closable"
+    :toolbar-items="bladeToolbar" @close="$emit('close:blade')" @expand="$emit('expand:blade')"
+    @collapse="$emit('collapse:blade')">
+
+    <!-- @vue-generic {never} -->
+    <VcTable :expanded="expanded" class="tw-grow tw-basis-0" multiselect :loading="loadingNewsArticles" :columns="columns"
+      :sort="searchQuery.sort" :pages="pagesCount" :total-count="newsArticlesCount" :search-value="searchKeyword"
+      :current-page="pageIndex" :search-placeholder="$t('VC_NEWS.PAGES.LIST.SEARCH.PLACEHOLDER')"
+      :total-label="$t('VC_NEWS.PAGES.LIST.TABLE.TOTALS')" :selected-item-id="selectedItemId" state-key="VC_NEWS"
+      :items="newsArticles" @item-click="onItemClick" @header-click="onHeaderClick" @pagination-click="onPaginationClick"
+      @search:change="onSearchChange" @selection-changed="onSelectionChanged">
+    </VcTable>
+  </VcBlade>
+</template>
+
+<script lang="ts" setup>
+import { computed, ref, markRaw, onMounted, watch } from "vue";
+import { IBladeEvent, IBladeToolbar, IParentCallArgs, ITableColumns, useBladeNavigation, usePopup } from "@vc-shell/framework";
+import { useI18n } from "vue-i18n";
+import { useNewsArticleList } from "../composables";
+import NewsArticleDetails from "./news-article-details.vue";
+import { NewsArticle } from "src/api_client/virtocommerce.news";
+
+export interface Props {
+  expanded?: boolean;
+  closable?: boolean;
+  param?: string;
+}
+
+export interface Emits {
+  (event: "parent:call", args: IParentCallArgs): void;
+  (event: "collapse:blade"): void;
+  (event: "expand:blade"): void;
+  (event: "open:blade", blade: IBladeEvent): void;
+  (event: "close:blade"): void;
+}
+
+defineOptions({
+  url: "/vc-news",
+  name: "NewsArticleList",
+  isWorkspace: true,
+  menuItem: {
+    title: "VC_NEWS.MENU.TITLE",
+    icon: "lucide-file",
+    priority: 1,
+  },
+});
+
+const props = withDefaults(defineProps<Props>(), {
+  expanded: true,
+  closable: true,
+});
+
+defineEmits<Emits>();
+
+const { showConfirmation } = usePopup();
+
+const { t } = useI18n({ useScope: "global" });
+const { openBlade } = useBladeNavigation();
+
+const { newsArticles, newsArticlesCount, pagesCount, pageIndex, searchNewsArticles, searchQuery, loadingNewsArticles, deleteNewsArticles } = useNewsArticleList();
+
+const searchKeyword = ref();
+const selectedItemId = ref<string>();
+const selectedIds = ref<string[]>([]);
+
+watch(
+  () => props.param,
+  (newVal) => {
+    selectedItemId.value = newVal;
+  },
+  { immediate: true },
+);
+
+const bladeToolbar = computed((): IBladeToolbar[] => [
+  {
+    id: "refresh",
+    title: computed(() => t("VC_NEWS.PAGES.LIST.TOOLBAR.REFRESH")),
```

Comment:
Your blade toolbar is already a computed property, so there's no point in using computed again inside it for the titles.